ChaosMod/Effects/db/Vehs/VehsGtaRacing.cpp

```diff
@@ -0,0 +1,22 @@
+#include <stdafx.h>
+
+//Effect by ProfessorBiddle, suggested by JustTunerYT
+
+static void OnTick()
+{
+	static auto lastTick = GetTickCount64();
+	auto curTick = GetTickCount64();
```

Comment:
You should switch to "GET_GAME_TIMER". For more info on the differences see https://github.com/gta-chaos-mod/ChaosModV/pull/202